primitives/core/Cargo.toml

```diff
@@ -25,7 +22,7 @@ primitive-types = { version = "0.12.0", default-features = false, features = ["c
 impl-serde = { version = "0.4.0", optional = true }
 hash-db = { version = "0.16.0", default-features = false }
 hash256-std-hasher = { version = "0.15.2", default-features = false }
-base58 = { version = "0.2.0", optional = true }
+bs58 = { version = "0.4.0", optional = true }
```

Comment:
Shouldn't we add `default-features = false` to disable `bs58`'s `std` feature and add it below in `#[features]`?

Reply:
It is `no_std` by default. Their `std` feature is actually just `alloc`

Reply:
> It is `no_std` by default. Their `std` feature is actually just `alloc`

From what I can see in the code it isn't? Unless I'm misunderstanding something.

e.g. from https://github.com/Nullus157/bs58-rs/blob/prīmum/src/lib.rs

```rust
#[cfg(feature = "std")]
extern crate std;
```

and then in https://github.com/Nullus157/bs58-rs/blob/prīmum/src/encode.rs

```rust
#[cfg(feature = "std")]
impl std::error::Error for Error {}
```

Reply:
Okay weird, I had only checked the `lib.rs` file :shrug: https://github.com/Nullus157/bs58-rs/blob/pr%C4%ABmum/src/lib.rs#L1